File: 2114.句子中的最多单词数.js
/*
 * @lc app=leetcode.cn id=2114 lang=javascript
 *
 * [2114] 句子中的最多单词数
 */

// @lc code=start
/**
 * @param {string[]} sentences
 * @return {number}
 */
var mostWordsFound = function(sentences) {
    let res = 0;
    for (let sentence of sentences) {
        let words = sentence.split(" ");
        res = Math.max(res, words.length);
    }
    return res;
};
// @lc code=end

File: 2114.句子中的最多单词数.java
/*
 * @lc app=leetcode.cn id=2114 lang=java
 *
 * [2114] 句子中的最多单词数
 */

// @lc code=start
class Solution {
    public int mostWordsFound(String[] sentences) {
        int res = 0;
        for (String sentence : sentences) {
            // 分割
            String[] words = sentence.split(" ");
            // 更新最大单词数
            res = Math.max(res, words.length);
        }
        return res;
    }
}
// @lc code=end

